src/common/htmlFileReader.ts

```diff
@@ -86,21 +86,24 @@ export default class HtmlFileReader {
             }
             data = await response.data;
         }
-        console.log("blobbing");
+
         return data;
     }
 
     public static async getAssetFrameImage(asset: IAsset) {
         return new Promise((resolve, reject) => {
-            let video: HTMLVideoElement;
-            let refresh = true;
-            if (asset.parent.name in this.videoAssetFiles) {
-                video = this.videoAssetFiles[asset.parent.name];
-                refresh = false;
-            } else {
+            const cachingEnabled = false;
```

Comment:
How does `cachingEnabled` ever get set to true?

Reply:
So right now it doesn't get reset, it's just for development as a toggle while we see how we can add in the caching, but for now the exporting works fine without the caching since the batching can help optimize things